applications/adv_networking_bench/cpp/default_bench_op_rx.h

```diff
@@ -186,185 +183,190 @@ class AdvNetworkingBenchDefaultRxOp : public Operator {
     free_processed_packets();
 
     BurstParams *burst;
-    auto status = get_rx_burst(&burst, port_id_, queue_ids_[0]);
-    if (status != Status::SUCCESS) {
-      HOLOSCAN_LOG_DEBUG("No RX burst available");
-      return;
-    }
 
-    auto burst_size = get_num_packets(burst);
+    const auto num_rx_queues = get_num_rx_queues(port_id_);
```

Comment:
Can we add a comment above similar to this?

```suggestion
    // In this example, we'll loop through all the rx queues of the interface
    // assuming we want to process the packets the same way for all queues
    const auto num_rx_queues = get_num_rx_queues(port_id_);
```